tokenizers/src/pre_tokenizers/byte_level.rs

```diff
@@ -53,22 +53,33 @@ pub struct ByteLevel {
     pub add_prefix_space: bool,
     /// Whether the post processing step should trim offsets to avoid including whitespaces.
     pub trim_offsets: bool,
+
+    /// Whether to use the standard GPT2 regex for whitespace splitting
+    /// Set it to False if you want to use you're own splitting.
+    #[serde(default = "default_true")]
+    pub use_regex: bool,
+}
+
+fn default_true() -> bool {
+    true
 }
```

Comment:
Hum, why do you need a function here ? Is it because the serde macro requires a  function ?

Reply:
Yes basically. I couldn't find another way to use the serde_default.